github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -500,8 +500,8 @@ presubmits:
     annotations:
       fork-per-release: "true"
     always_run: true
```

Comment:
Let’s make it on demand lane since it doesn’t report and doesn’t block the merge anyway.

Reply:
We want to enable it shortly again, collected information about failed runs would be helpful for debugging.

Reply:
The errors are due to prow problems, not because of ipv6 lane specific problems, i would suggest not to make changes to this lane.